### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by RichardIvan**
Explains how to mock the dom for mithril query to run without errors

### CODE REVIEW

**Comment by magnusleo** [comment]
+1

**Comment by magnusleo** [comment]
Maybe fix the example files too.

File: README.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ Installation
...
+ 
+Setup
+-----
+
+In order to run tests in mithril 1.0 we need to do some setup. That is to mock the dom for the mithril render and request modules.
+This can be done by requiring a 'setup' file in your 'mocha' tests with the following contents.
+
+```   
+global.window = require('mithril/test-utils/domMock.js')()
+global.window = Object.assign(global.window, require('mithril/test-utils/pushStateMock')())
```
Comment (barneycarroll) [inline README.md]: The 2 lines above could be written as:

```js
global.window = Object.assign(require('mithril/test-utils/domMock.js')(), require('mithril/test-utils/pushStateMock')())
```

**Comment by StephanHoyer** [comment]
What do you think of adding a method to mq to make this easier like

```
mq.mockBrowser()
```

**Comment by magnusleo** [comment]
That method would be helpful.

**Pull request merged**